pkgs/by-name/al/alive2/package.nix

```
@@ -8,16 +8,17 @@
   llvm_18,
   cmake,
   ninja,
+  nix-update-script,
 }:
 
 clangStdenv.mkDerivation (finalAttrs: {
   pname = "alive2";
-  version = "0-unstable-2024-09-23";
+  version = "20.0";
 
   src = fetchFromGitHub {
     owner = "AliveToolkit";
     repo = "alive2";
-    rev = "05a964284056b38a6dc1f807e7acad64a0308328";
+    tag = "v${finalAttrs.version}";
     sha256 = "sha256-okKKUU7WLXLD9Hvsfoz+1HQWoyQ/bqRpBk5ogr7kSJA=";
```

The hashes also need to be updated when the version is changed:

```suggestion
    hash = "sha256-4QNrBRGH+rxXwb7zTRYAixxipN3ybcXuWCmO+BLU9r4=";
```